src/pyglottolog/api.py

```diff
@@ -25,6 +26,30 @@
 ISO_CODE_PATTERN = re.compile('[a-z]{3}$')
 
 
+class Cache:
+    def __init__(self):
+        self.by_iso = {}
```

Comment:
Could stick with a single dictionary as paths and iso-codes dont overlap (not sure if we want to expose the dicts here).

Maybe implement `__len__` (e.g. for testing)?